app/views/publishers/_paypal_account_panel.html.slim

```diff
@@ -0,0 +1,59 @@
+.dashboard-panel--wrapper
+  .dashboard-panel--header-with-image
+    .dashboard-panel--header-image
+      = image_tag("wallet@2x.png", width: 30, height: 30)
+    = t ".heading"
+    .dashboard-panel--header-right-aligned-image
+      = image_tag("paypal-logo.jpg", width: 55, height: 55)
+  .dashboard-panel--content#uphold_status_display
+    .deposit-date-explanation
+      = t ".deposit_date_explanation"
+    - if @publisher.paypal_connection.present?
+      .d-flex.justify-content-between
+        .payout-connected-highlight
+          span &#9679;&nbsp
+          = t(".connected")
+          span &nbsp&#8226;&nbsp
+          = @publisher.paypal_connection.email
+        .payout-account-disconnect-button
+          = link_to t(".disconnect"), paypal_account_disconnect_path(@publisher.paypal_connection.id)
+      br
+  - if @publisher.paypal_connection.present? 
+    - if @publisher.paypal_connection.verified_account?
+      .dashboard-panel--content#last_settlement class=last_settlement_class(@publisher)
+        .last-deposit-v2.d-flex.justify-content-between
+          .last-deposit-v2--label
+            = t ".last_deposit"
+          .d-flex.justify-content-start.flex-column
+            - last_deposit_balance = publisher_last_settlement_bat_balance(@publisher)
+            - if last_deposit_balance == I18n.t("helpers.publisher.no_deposit")
+              div.last-deposit-v2--date
+                label = t(".no_deposit_made_yet")
+            - else
+              div.last-deposit-v2--amount
+                span = last_deposit_balance + " " + t(".bat_points")
+              div.last-deposit-v2--date
+                label = t(".last_deposit_date") + publisher_last_settlement_date(@publisher, params[:locale])
+    - else
+      .dashboard-panel--content#last_settlement
+        .last-deposit-v2.last-deposit-v2--no-payout-account
+          = t ".need_bank_account"
+  .wide-action
+    - if @publisher.paypal_connection.nil?
+      = link_to t(".connect_with_paypal"),
+        "#{Rails.application.secrets[:paypal_api_uri]}/connect?flowEntry=static&client_id=#{Rails.application.secrets[:paypal_client_id]}&scope=openid email address https%3A%2F%2Furi.paypal.com%2Fservices%2Fpaypalattributes&redirect_uri=https%3A%2F%2Fbat-publishers-dev.herokuapp.com%2Fpublishers%2Fconnect_callback",
```

Comment:
I think this should be in a helper, it looks very strange in a view.

Also concerned that I see `bat-publishers-dev.herokuapp.com` the callback URL as this value will change between environments